members/C001088.yaml

```diff
@@ -164,13 +163,13 @@ contact_form:
           Trade: Trade
           Transportation and Infrastructure: Transportation and Infrastructure
           Veterans: Veterans
-          Women's Issues: Women's Issues
-          Other: Other
+          Miscellaneous: Miscellaneous
+          National Service: National Service
     - click_on:
       - value: Submit
-        selector: form#B03E03FC-4040-F985-52CD-BCD7E00A8A6C div.controls input.btn
-    - wait:
-      - value: 3
+        selector: input[type='submit'][value='Submit']
+    - find:
+      - selector: div p:nth-child(2)
```

Comment:
can you describe how you chose the selector for this find?

Reply:
The two `p` elements in the success message page don't really have any class or id that sets them apart. Thought it was easier/more straightforward to use `:nth-child` to select the appropriate selector here. 

[![Screenshot from Gyazo](https://gyazo.com/338616da2589b32e2a71240f21f53bf4/raw)](https://gyazo.com/338616da2589b32e2a71240f21f53bf4)

Reply:
👍